presto-verifier/src/main/java/com/facebook/presto/verifier/framework/VerificationFactory.java

```diff
@@ -100,18 +100,34 @@ public Verification get(SourceQuery sourceQuery,
         }
 
         QueryRewriter queryRewriter = queryRewriterFactory.create(queryActions.getHelperAction());
+        DeterminismAnalyzer determinismAnalyzer = new DeterminismAnalyzer(
+                sourceQuery,
+                queryActions.getHelperAction(),
+                queryRewriter,
+                checksumValidator,
+                typeManager,
+                determinismAnalyzerConfig);
+        FailureResolverManager failureResolverManager = failureResolverManagerFactory.create(new FailureResolverFactoryContext(sqlParser, queryActions.getHelperAction()));
```

Comment:
Before this change we only ran this for "QUERY" (meaning SELECT I assume).
Now, having it in front of the switch, we run it for CREATE_VIEW and CREATE_TABLE as well.
Is this intended?

Reply:
Otherwise we need to create them for [CREATE_TABLE_AS_SELECT, INSERT] and [QUERY] separately and the code looks wordy and a bit messy.
Those three lines only creates object and has no external/expensive calls, and it makes the code much cleaner.

Reply:
previous logic falls thru all 3 query types CTAS, INSERT, QUERY, using same code path, which is ok.
but for below CREATE_VIEW and CREATE_TABLE, it seems not necessary.